openapi-contracts/src/main/com/sinch/sdk/domains/conversation/models/dto/v1/AppMessageTypesDto.java

```diff
@@ -0,0 +1,636 @@
+/*
+ * Conversation API | Sinch
+ * Send and receive messages globally over SMS, RCS, WhatsApp, Viber Business, Facebook messenger and other popular channels using the Sinch Conversation API.  The Conversation API endpoint uses built-in transcoding to give you the power of conversation across all supported channels and, if required, full control over channel specific features.
+ *
+ * The version of the OpenAPI document: 1.0
+ * Contact: support@sinch.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sinch.sdk.domains.conversation.models.dto.v1;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.sinch.sdk.core.models.AbstractOpenApiSchema;
+import com.sinch.sdk.core.utils.databind.JSONNavigator;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@JsonDeserialize(using = AppMessageTypesDto.AppMessageTypesDtoDeserializer.class)
+@JsonSerialize(using = AppMessageTypesDto.AppMessageTypesDtoSerializer.class)
+public class AppMessageTypesDto extends AbstractOpenApiSchema {
+  private static final Logger log = Logger.getLogger(AppMessageTypesDto.class.getName());
+
+  public static class AppMessageTypesDtoSerializer extends StdSerializer<AppMessageTypesDto> {
+    public AppMessageTypesDtoSerializer(Class<AppMessageTypesDto> t) {
+      super(t);
+    }
+
+    public AppMessageTypesDtoSerializer() {
+      this(null);
+    }
+
+    @Override
+    public void serialize(AppMessageTypesDto value, JsonGenerator jgen, SerializerProvider provider)
+        throws IOException, JsonProcessingException {
+      jgen.writeObject(value.getActualInstance());
+    }
+  }
+
+  public static class AppMessageTypesDtoDeserializer extends StdDeserializer<AppMessageTypesDto> {
+    public AppMessageTypesDtoDeserializer() {
+      this(AppMessageTypesDto.class);
+    }
+
+    public AppMessageTypesDtoDeserializer(Class<?> vc) {
+      super(vc);
+    }
+
+    @Override
+    public AppMessageTypesDto deserialize(JsonParser jp, DeserializationContext ctxt)
```

Comment:
I trust the generator to do wise things here

Reply:
🤞